src/fis/adapters/outbound/vault/client.py

```diff
@@ -57,6 +57,11 @@ class VaultConfig(BaseSettings):
         description="Path without leading or trailing slashes where secrets should"
         + " be stored in the vault.",
     )
+    vault_secrets_mount_point: str = Field(
+        default="secret",
+        examples=["secret"],
```

Comment:
```suggestion
        default="file-secrets",
        examples=["file-secrets"],
```
`secret` is a bit too generic, as everything we store is a secret.
Not sure if `file_secrets` might be better or not

Reply:
I think the same. However, the default mount point for the HashiCorp Vault KV2 engine is `secret`. So I think we should leave it that way.

Reply:
So the default case would be backwards compatible with what we have right now?